import.py

```diff
@@ -73,25 +91,29 @@ def main():
       adminSiteRegistrations += "admin.site.register(" + stem + ", GeoNoEditAdmin)\n"
 
       loadMappings += stem + "_mapping = {\n"
-      loadMappings += "    '" + keyField.lower() + "': '" + keyField + "',\n"
+      if shapefileFound:
+        loadMappings += "    '" + keyField.lower() + "': '" + keyField + "',\n"
       loadMappings += "    'geom': '" + shapeType.upper() + "'\n"
       loadMappings += "}\n\n"
-      loadPaths += stem + "_shp = " + "os.path.abspath(os.path.join(os.path.dirname(__file__)," + " '../" + simplified + "'))\n"
+      if shapefileFound:
+        loadPaths += stem + "_shp = " + "os.path.abspath(os.path.join(os.path.dirname(__file__)," + " '../" + simplified + "'))\n"
+      elif rasterFound:
+        loadPaths += stem + "_tif = " + "os.path.abspath(os.path.join(os.path.dirname(__file__)," + " '../" + os.path.join(dataDir, f) + "'))\n"
       loadImports += "    print('Loading data for " + stem + "')\n"
       loadImports += "    from .models import " + stem + "\n"
-      loadImports += "    lm_" + stem + " = LayerMapping(" + stem + ", " + stem + "_shp, " + stem + "_mapping, transform=True, " + "encoding='" + encoding + "', unique=['" + keyField.lower() + "'])\n"
-      loadImports += "    lm_" + stem + ".save(strict=True, verbose=verbose)\n\n"
+      if shapefileFound:
+        loadImports += "    lm_" + stem + " = LayerMapping(" + stem + ", " + stem + "_shp, " + stem + "_mapping, transform=True, " + "encoding='" + encoding + "', unique=['" + keyField.lower() + "'])\n"
+      elif rasterFound:
+        loadImports += "    " + stem + "_raster = GDALRaster(" + stem + "_tif, write=True)\n"
+        loadImports += "    lm_" + stem + " = " + stem + "(rast=" + stem + "_raster)\n"
+      loadImports += "    lm_" + stem + ".save()\n\n"
 
       print("")
       first = False
 
   # clear trailing comma from this one
   modelsLocationsList = modelsLocationsList.strip(",\n") + "\n"
 
-  # assemble the whole return statement for the snugget class after going through the loop
-  modelsSnuggetReturns = "        return groupsDict\n"
```

Comment:
This was duplicating a line that is already in the skeleton models.py (line 359).  The duplication didn't seem to be doing any harm, but removing it changes no behaviour.